.gitignore

```diff
@@ -0,0 +1,5 @@
+*.env
+node_modules/
+.DS_Store
+compiled/
+client/dist/main.js
```

Comment:
sorry :) can we add a blank line to the end of this file too?

Reply:
All set!